Too many files

Deployment to Cloudflare is failing because we have too many files.

```
✘ [ERROR] Error: Pages only supports up to 20,000 files in a deployment. Ensure you have specified your build output directory correctly.
```

--- a/.github/workflows/pretext-cli.yml
+++ b/.github/workflows/pretext-cli.yml
@@ -56,7 +56,7 @@ jobs:
     deploy-cloudflare:
         runs-on: ubuntu-latest
         needs: build
-        if: vars.CLOUDFLARE_PROJECT_NAME != ''
+        if: vars.CLOUDFLARE_PROJECT_NAME != '' && github.ref != format('refs/heads/{0}', github.event.repository.default_branch)
         permissions:
           contents: read
           deployments: write

